src/ModelObserver.php

```diff
@@ -32,7 +32,7 @@ public function __construct(DynamoDbClientInterface $dynamoDb)
         $this->attributeFilter = $dynamoDb->getAttributeFilter();
     }
 
-    public function saved($model)
+    protected function _save($model)
```

Comment:
I would prefer to stay away from underscore prefix in method name. Could you:

1. Change this to `private`
2. Rename it to `saveToDynamoDb`

Same for the `_delete` method.